hicommonwealth/commonwealth
# Add community terms of service support

This pull adds community support for a TOS URI, which community admins can set in the community management modal.

This URI is then displayed whenever a user attempts to join a community with a new address, or accepts an email-based community invite.

It does not provide support for silent address-based invitations; see issue #1267. 

To test:
1. From an admin account, add a TOS link (should be valid http/s or will fail) to the community metadata form, in the "Manage Community" modal.
2. Add a new address to the community via the "Manage addresses"/SelectNewAddress modal
3. From an admin account, invite an email account (e.g. via mail.tm) to the community, then accept this invite by adding/connecting a new address to the community.

At each stage in which a user is connecting a new address to join the community, a link to the TOS should render. These steps should replicate on both on- and off-chain, and both public and private, communities.
